packages/next/src/views/Document/getDocumentData.ts

```diff
@@ -31,6 +40,11 @@ export const getDocumentData = async ({
         fallbackLocale: false,
         locale: locale?.code,
         overrideAccess: false,
+        req: {
```

Comment:
Feels odd to construct a new req object here instead of passing it through outright. I imagine this is to reduce the size of the actual request as much as possible. Pretty sure you wouldn't have to send `user` as a stand-alone property in this case.

Reply:
> imagine this is to reduce the size of the actual request as much as possible

Doesn't matter since this all runs on the server anyways.

I mainly wanted to avoid the overhead of spreading `req` in order to remove the `transactionID`

Reply:
Makes sense